packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -617,16 +617,13 @@ export class Relayer {
       }
     | undefined
   > {
-    const mrkdwn = this._generateMarkdownForRelay(relayableDeposit.deposit, realizedLpFeePct);
-    switch (shouldRelay) {
+    const mrkdwn = this._generateMarkdownForRelay(relayableDeposit.deposit, realizedLpFeePct, profitabilityInformation);
+    switch (relaySubmitType) {
       case RelaySubmitType.Ignore:
         this.logger.warn({
```

Comment:
next PR will add logic to not re-generate this log if it was previously produced.